geomesa-hbase/geomesa-hbase-datastore/README.md

```diff
@@ -0,0 +1,21 @@
+# GeoMesa HBase Data Store
+
+The data store module contains all of the HBase-related code for GeoMesa.
+ 
+## HBase Data Store
+
+An instance of an HBase data store can be obtained through the normal GeoTools discovery methods, assuming that the GeoMesa code is on the classpath. The HBase data store also requires that an `hbase-site.xml` be located in the classpath root; the connection parameters for the HBase data store, including `hbase.zookeeper.quorum` and `hbase.zookeeper.property.clientPort`, are obtained from this file.
```

Comment:
[mind-bogglingly picky]  Is there such a thing as the "classpath root", or do we simply mean that the configuration file needs to be either in a JAR on the classpath or in the root of any directory on the classpath?